FrontEnd/src/components/SearchPage/Search.js

```diff
@@ -0,0 +1,142 @@
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import BreadCrumbs from '../BreadCrumbs/BreadCrumbs';
+import SearchResults from './search_field/SearchResults';
+import frame42 from './img/frame42.png';
+import link_to_left from './img/link_to_left.svg';
+import link_to_right from './img/link_to_right.svg';
+import './search_page.css';
+
+const ITEMS_PER_PAGE = 6;
+
+export function Search(props) {
+  const [searchResults, setSearchResults] = useState([]);
+  const [searchPerformed, setSearchPerformed] = useState(false);
+  const [error, setError] = useState(null);
+
+  const location = useLocation();
+  const searchParams = new URLSearchParams(location.search);
+  const searchTerm = searchParams.get('name');
+  const servedAddress = process.env.REACT_APP_BASE_API_URL;
+  const searchUrl = 'search';
+
+  useEffect(() => {
+    if (searchTerm) {
+      // Make an AJAX request to Django API to get search results
+      axios
+        .get(`${servedAddress}/api/search/?name=${searchTerm}`)
+        .then((response) => {
+          setSearchResults(response.data);
+          setSearchPerformed(true);
+          setError(null); // Clear error on successful response
+        })
+        .catch((error) => {
+          console.error('Error fetching search results:', error);
+          setError(error.response ? error.response.data : 'An error occurred');
+        });
```

Comment:
I think we can handle it using swr for the consistency

Reply:
Thank you for feedback. Done